promtail-config.yaml

```
@@ -16,4 +16,4 @@ scrape_configs:
         labels:
           job: varlogs
           compose_service: app-universal
-          __path__: /app/logs/*.log
+          __path__: /app/logs/*.log
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)